front/app/containers/HomePage/SignedOutHeader/HeaderContent.tsx

```diff
@@ -102,7 +102,9 @@ const HeaderContent = ({
 
         {displayHeaderAvatars && <StyledAvatarBubbles />}
 
-        <CTA signedIn={false} buttonStyle={buttonStyle} signUpIn={signUpIn} />
+        <div style={{ marginBottom: '20px' }}>
```

Comment:
Not related to tests but let's avoid using inline styles in this way. We have established mechanisms - a `Box` component that provides this functionality and we can always use a style component, if something more is needed.